bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -161,6 +161,34 @@ spec:
       requests:
         memory: 50Mi
         cpu: 10m
+  - name: kube-apiserver-check-endpoints
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: Always
```

Comment:
IfNotPresent.  Also, this is worth a bug or jira to write an e2e test for at some later date.  cc @sttts

Reply:
test for what?

Reply:
@deads2k  I think there actually is one, I've seen it fail.